server/src/main/webapp/WEB-INF/rails/webpack/helpers/spark_routes.ts

```diff
@@ -439,4 +439,8 @@ export class SparkRoutes {
   static editTemplatePermissions(templateName: string) {
     return `/go/admin/templates/${templateName}/permissions`;
   }
+
+  static editEnvironmentsPath(environmentName: string) {
```

Comment:
is this an `editEnvironmentsPath` ? or a `getEnvironmentPathOnSPA`?

Reply:
Changed as the part of [this](https://github.com/gocd/gocd/pull/7320/commits/aaac131674ca254d41d8957a7545bb3beb5d7c01) commit.